src/dfx/src/lib/builders/mod.rs

```diff
@@ -345,8 +345,8 @@ pub fn get_and_write_environment_variables<'a>(
     for canister in pool.get_canister_list() {
         vars.push((
             Owned(format!(
-                "CANISTER_ID_{}",
-                canister.get_name().replace('-', "_")
+                "{}_CANISTER_ID",
+                canister.get_name().replace('-', "_").to_ascii_uppercase(),
```

Comment:
This looks like a breaking change for existing projects.

Two things:
- please add a migration guide here: https://github.com/dfinity/sdk/tree/master/docs/migration
- I think we should first alter this code to add _both_ forms (if we are going to rename this), and deprecate the old form, rather than deleting it right away.

Reply:
I don't think that we have released this yet. This is a new feature, and I noticed it wasn't compatible with the existing declarations we have been shipping, both because `CANISTER_ID` was being used as a prefix instead of suffix, and also because the canister name was lowercase.